include/deal.II/base/mpi.h

```diff
@@ -491,7 +491,7 @@ namespace Utilities
     bool job_supports_mpi ();
 
     /**
-     * Initiate a some-to-some communication, and exchange arbitrary objects
+     * Initiate a some-to-some communication, and exchanges arbitrary objects
```

Comment:
If you want `exchanges` you should also use `Initiates`.

Reply:
Right -- it was apparently still too early in the morning...

Fixed now.